feature/platform/tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -759,6 +762,14 @@ func ValidateComponentState(t *testing.T, dut *ondatra.DUTDevice, cards []*oc.Co
 				}
 			}
 
+			if p.modelNameValidation {
+				if deviations.ModelNameUnsupported(dut) {
+					t.Logf("Telemetry path /components/component/state/model-name is not supported due to deviation ModelNameUnsupported. Skipping model name validation.")
+				} else if card.GetModelName() != dut.Model() {
```

Comment:
dut.Model() will reflect the value from ondatra binding, but it is not guaranteed to be an exact match of the leaf value. 

I don't think this is needed

Reply:
My work will depend on the chassis' `components/component/state/model-name` being the correct model name for that device, so I would like to try and test this if possible so that vendors have the correct value in this leaf. 

Is this name stored anywhere else then so that I could validate this? Or were you suggesting more along the lines of just checking that it is non-empty when expected to validate it?

Reply:
>being the correct model name for that device, so I would like to try and test this if possible so that vendors have the correct value in this leaf.

The question is, how the correct value is defined?

For example, if I have a device "7250 IXR-10e"; and in your environment dut.Model() returns something like `7250-ixr-10e` or `ixr-10e` or any other combination of the words, the test will fail, but not because the model-name is incorrect. 

Of course, it can be fixed, but the action will be to update the names provisioned in lab infra.

>Or were you suggesting more along the lines of just checking that it is non-empty when expected to validate it?

Yep. I wouldn't object either way, just wanted to highlight the above point.

Reply:
My understanding is that the vendor will need to decide on a canonical hardware model name and then assign this to the `components/component/state/model-name` leaf. I will then be using this leaf as the source of truth to determine which device model it is. And yes they would also need to make sure the dut.Model() matches this value.

@dplore does that make sense?

Reply:
We should validate with the ondatra binding.  Agreed that the maintainer of the binding may need to update the value to get it to match.  ondatra binding seems like a good place to assert what the expected model name should be.  That ondatra model name could be used in many other places in the tests, and not just this gnmi-1.4 test.

Reply:
WDYT @LimeHat ?   Returning any value is somewhat useful to test, but we have operational use cases that depend on an exact match with an expected value.  I think I'm happy with ensuring the ondatra binding contains the expected value.

Reply:
Sounds good to me.